tools/php-test-env/composer.json

```diff
@@ -0,0 +1,38 @@
+{
+	"name": "automattic/jetpack-test-environment",
+	"description": "Jetpack Test Environment Helpers",
+	"type": "library",
+	"license": "GPL-2.0-or-later",
+	"require": {
+		"php": ">=7.0",
+		"automattic/wordbless": "^0.4.2",
+		"yoast/phpunit-polyfills": "^1.1.3"
+	},
+	"require-dev": {
+		"automattic/jetpack-changelogger": "@dev",
+		"automattic/jetpack-codesniffer": "@dev"
+	},
+	"scripts": {
+		"post-install-cmd": "WorDBless\\Composer\\InstallDropin::copy",
+		"post-update-cmd": "WorDBless\\Composer\\InstallDropin::copy"
+	},
+	"minimum-stability": "dev",
+	"prefer-stable": true,
+	"extra": {
+		"autotagger": true,
+		"mirror-repo": "Automattic/jetpack-test-environment",
+		"changelogger": {
+			"link-template": "https://github.com/Automattic/jetpack-test-environment/compare/v${old}...v${new}"
+		},
+		"branch-alias": {
+			"dev-trunk": "0.2.x-dev"
+		}
+	},
+	"config": {
+		"sort-packages": true,
```

Comment:
We may want to add [`"prepend-autoloader": false,`](https://getcomposer.org/doc/06-config.md#prepend-autoloader) in here.

As is, because `tools/php-test-env/vendor/autoload.php` is likely to be loaded late, anything it provides is likely to unexpectedly override whatever the package itself tries to provide. That might mean, for example, a different version of PHPUnit.

Setting `"prepend-autoloader": false,` avoids that by having Composer register its autoloader after all other already-registered autoloaders, rather than before them all.

Reply:
ah, that makes sense. adding that.